core/server/master/src/main/java/alluxio/master/AlluxioMasterProcess.java

```diff
@@ -305,6 +305,16 @@ protected void startServingRPCServer() {
           GrpcServerAddress.create(mRpcConnectAddress.getHostName(), mRpcBindAddress),
           ServerConfiguration.global(), ServerUserState.global());
 
+      int parallelism = ServerConfiguration.getInt(PropertyKey.MASTER_RPC_EXECUTOR_PARALLELISM);
+      Preconditions.checkArgument(parallelism > 0,
+              String.format("Starting Alluxio master gRPC thread pool with %s executors! "
```

Comment:
It is not actually starting, so should it say something "Cannot start master gRPC thread pool with ..."?